src/components/chapters/ChapterHeader/index.tsx

```diff
@@ -1,31 +1,60 @@
-import Bismillah, { BismillahSize } from 'src/components/dls/Bismillah/Bismillah';
 import React from 'react';
+import Button, { ButtonSize, ButtonVariant } from 'src/components/dls/Button/Button';
+import Bismillah, { BismillahSize } from 'src/components/dls/Bismillah/Bismillah';
 import PlayChapterAudioButton from 'src/components/QuranReader/PlayChapterAudioButton';
-import Button, { ButtonVariant } from 'src/components/dls/Button/Button';
+import { formatChapterId, getChapterInfoUrl } from 'src/utils/verse';
 import ChapterIconContainer, { ChapterIconsSize } from '../ChapterIcon/ChapterIconContainer';
 import styles from './ChapterHeader.module.scss';
+import QOutlineIcon from '../../../../public/icons/Q-outline.svg';
+import InfoIcon from '../../../../public/icons/info.svg';
 
 interface Props {
   chapterId: string;
+  nameSimple: string;
+  translatedName?: string;
```

Comment:
either we should require this e.g. `translatedName: string;` or check if it exists inside the component and only show it when it does.